media/plg_media-action_smartcrop/js/smartcrop.js

```diff
@@ -79,11 +85,39 @@ Joomla.MediaManager.Edit = Joomla.MediaManager.Edit || {};
 		return basePath;
 	}
 
+	function addCustomWidths() {
+		var widths = Joomla.getOptions('js-smartcrop-widths');
+		var widthDropDown = document.getElementById("jform_requestedWidth");
+		var defaultWidthLength = widthDropDown.length;
+		var customWidthLength  = widths.length;
+		if (customWidthLength > 0)
+		{
```

Comment:
can you follow a same style like

```
if () {
} 
```

you are mixing two styles here. just use one style